### PR TITLE
FF118 mathvariant not global

### PR DESCRIPTION
**Pull request by hamishwillee**
The MathML [`mathvariant`](https://developer.mozilla.org/en-US/docs/Web/MathML/Global_attributes/mathvariant) is documented as a global attribute that can be used on many/several elements (but primarily [`<mi>`](https://developer.mozilla.org/en-US/docs/Web/MathML/Element/mi)). This is according to the MathML 3 spec. 

Firefox and other browsers are now moving towards the MathML core spec which redefines [`mathvariant`](https://developer.mozilla.org/en-US/docs/Web/MathML/Global_attributes/mathvariant) as an attribute **only** of [`<mi>`](https://developer.mozilla.org/en-US/docs/Web/MathML/Element/mi), and only allows one value: 'normal'. 

FF118 is a aligning with this in https://bugzilla.mozilla.org/show_bug.cgi?id=1845461 by explicitly deprecating use on other elements and of values other than 'normal'. Chrome has already effectively done this by only allowing 'normal'.

This PR moves `mathvariant` from a global attribute to be under `<mi>` and rewrites it as per the current spec. The old behaviour is still noted, as an explanation of the deprecated attributes.

This is part of https://github.com/mdn/content/issues/28849

### CODE REVIEW

File: files/en-us/web/mathml/element/mi/index.md [hunk 1]
```diff
@@ -7,32 +7,89 @@ browser-compat: mathml.elements.mi
 
 {{MathMLRef}}
 
-The **`<mi>`** [MathML](/en-US/docs/Web/MathML) element indicates that the content should be rendered as an **identifier** such as function names, variables or symbolic constants. You can also have arbitrary text in it to mark up terms.
+The **`<mi>`** [MathML](/en-US/docs/Web/MathML) element indicates that the content should be rendered as an **identifier**, such as a function name, variable or symbolic constant.
+
+By default `<mi>` elements that contain multiple characters are a rendered as normal text, while single character characters are rendered as italic: the same formatting behaviour as the [CSS `text-transform`](/en-US/docs/Web/CSS/text-transform) property with a value of `math-auto`.
+The `mathvariant` attribute with a value of `normal` can be used to reset a single character to the normal font.
+
+In order to use a particular form of a character such as bold/italic, serif, san-serif, script/calligraphy, monospaced, double-struck, and so on, you should use the appropriate [Mathematical Alphanumeric Symbols](https://en.wikipedia.org/wiki/Mathematical_Alphanumeric_Symbols).
```
Comment (fred-wang) [inline files/en-us/web/mathml/element/mi/index.md:15]: sans-serif*

File: files/en-us/web/mathml/element/mi/index.md [hunk 2]
```diff
@@ -7,32 +7,89 @@ browser-compat: mathml.elements.mi
...
-  <!-- Multiple characters, default mathvariant is "normal". -->
+  <!-- Multiple characters render as "normal" text -->
   <mi>sin</mi>
 </math>
-
+<hr />
 <math display="block">
-  <!-- Single character, default mathvariant is "italic". -->
-  <mi>y</mi>
+  <!-- Single characters render as italic by default (i.e. "y" renders as "𝐴") -->
```
Comment (fred-wang) [inline files/en-us/web/mathml/element/mi/index.md:77]: s/y/A/

**Review by fred-wang — APPROVED** [review]
Thanks!

File: files/en-us/web/mathml/element/mi/index.md (hunk 1 above)
Comment (hamishwillee) [reply files/en-us/web/mathml/element/mi/index.md:15]:
```suggestion
In order to use a particular form of a character such as bold/italic, serif, sans-serif, script/calligraphy, monospaced, double-struck, and so on, you should use the appropriate [Mathematical Alphanumeric Symbols](https://en.wikipedia.org/wiki/Mathematical_Alphanumeric_Symbols).
```

File: files/en-us/web/mathml/element/mi/index.md (hunk 2 above)
Comment (hamishwillee) [reply files/en-us/web/mathml/element/mi/index.md:77]:
```suggestion
  <!-- Single characters render as italic by default (i.e. "A" renders as "𝐴") -->
```

File: files/en-us/learn/mathml/first_steps/text_containers/index.md [hunk 3]
```diff
@@ -181,9 +181,14 @@ One typographic convention in mathematics is to use italic letters for variables
...
 ```
 
+{{ EmbedLiveSample('Automatic italicization of <mi>', 700, 50) }}
+
 > **Note:** [This table from MathML Core](https://w3c.github.io/mathml-core/#italic-mappings) provide the exhaustive list of characters that are subject to italicization, together with the corresponding italic characters.
 
-It is sometimes needed to revert this default italic transformation. For that purpose, just attach a `mathvariant="normal"` attribute on the `<mi>` element. Compare the rendering of the uppercase gamma letters in the following formula:
+## Reverting automatic italicization of \<mi>
+
+It is sometimes needed to revert this default italic transformation. For that purpose, just attach a `mathvariant="normal"` attribute on the `<mi>` element.
```
Comment (hamishwillee) [inline files/en-us/learn/mathml/first_steps/text_containers/index.md:190]:
```suggestion
In order to revert this default italic transformation you can attach a `mathvariant="normal"` attribute on the `<mi>` element.
```

**Pull request merged**